contracts/Aavegotchi/facets/ItemsFacet.sol

```diff
@@ -287,7 +290,7 @@ contract ItemsFacet is Modifiers {
 
                 if (nftBalance < neededBalance) {
                     //Transfer to Aavegotchi
-                    _addWearableToGotchi(_depositIds[slot], _tokenId, toEquipId, neededBalance, nftBalance);
+                    _addWearableToGotchi(_depositIds[slot], _tokenId, toEquipId, neededBalance - nftBalance, _sender, _equippedItemsInfo);
```

Comment:
I don't know if we should pass `_sender` as function parameters. If that were the case, we would need it on every single function that required `msg.sender`. What is the Diamond best practice?